spec/factories/users.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :user do
+    sequence(:first_name) { |n| "Test user #{n}" }
+    sequence(:email) { |n| "user#{n}@gmail.com" }
+    password { "abcdef" }
+    password_confirmation { "abcdef" }
+    confirmed_at { Time.current }
+    
+    trait :admin do
```

Comment:
This should be a factory in its own `admin_user` rather than being a trait as this contains more complex logic to create a user

Reply:
done